Apps/Sandcastle/gallery/Particle System.html

```diff
@@ -263,20 +263,16 @@
     startScale : viewModel.startScale,
     endScale : viewModel.endScale,
 
-    minimumLife : viewModel.minimumLife,
-    maximumLife : viewModel.maximumLife,
+    minimumParticleLife : viewModel.minimumParticleLife,
+    maximumParticleLife : viewModel.maximumParticleLife,
 
     minimumSpeed : viewModel.minimumSpeed,
     maximumSpeed : viewModel.maximumSpeed,
 
-    minimumWidth : viewModel.particleSize,
-    minimumHeight : viewModel.particleSize,
-
-    maximumWidth : viewModel.particleSize,
-    maximumHeight : viewModel.particleSize,
+    imageSize : new Cesium.Cartesian2(viewModel.particleSize, viewModel.particleSize),
```

Comment:
Using the slider for "Size" in the example throws errors. You'll need to have this observable be one number value, then update `imageSize` accordingly when update in the following function:

`Cesium.knockout.getObservable(viewModel, 'particleSize').subscribe(`

Reply:
done.